libs/features/components/cards/src/lib/features-components-cards.module.ts

```diff
@@ -0,0 +1,7 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+@NgModule({
+  imports: [CommonModule],
+})
+export class FeaturesComponentsCardsModule {}
```

Comment:
Rename to CardsModule

Reply:
Resolved the issue. Waiting for another review.